pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
```

Comment:
This file and these two lines are really important for Python projects. They tell the Python packaging ecosystem what build tool your project uses to build. Regardless of this PR, these two lines should be added.